### PR TITLE
Announce 2.7.2

### PR DESCRIPTION
**Pull request by jeff5**
Update links including installation page.

### CODE REVIEW

File: news.md [hunk 1]
```diff
@@ -3,6 +3,16 @@ title: News
...
+
+A new full release is now available for Jython 2.7.2 at
+[Maven Central](https://search.maven.org/search?q=g:org.python).
+It is built and tested with Java 8 and tested against Java 11.
+There are no changes (apart from the version identification) from v2.7.2rc1.
+
+Fixed bugs for all versions are listed in [NEWS](https://github.com/jythontools/jython/blob/v2.7.2/NEWS).
+
+
 ### Jython 2.7.2 release candidate (v2.7.2rc1 March 2020)
```
Comment (adamburkegh) [inline news.md]: It was March 5 if you want to retrofit this date

**Comment by jamesmudd** [comment]
Think I would reword line 13. How about "for a full list of bugs fixed see news"?

But otherwise looks great. Thanks for all your work on this.

File: news.md (hunk 1 above)
Comment (jeff5) [reply news.md]: Yeah, why not.

**Comment by jeff5** [comment]
I followed both suggestions.

I tried (this time around) to follow the instructions to test locally using Jekyll, but after huge installations all I got was incompatible dependencies and stack dumps (Linux and Windows). However, none of my changes is very adventurous.

It makes you realise how good pip is.
